src/argus/incident/filters.py

```diff
@@ -26,6 +28,30 @@ class IncidentFilter(filters.FilterSet):
     tags = TagInFilter(label="Tags", method="incident_filter")
     duration__gte = filters.NumberFilter(label="Duration", method="incident_filter")
     token_expiry = filters.BooleanFilter(label="Token expiry", method="incident_filter")
+    filter_pk = filters.NumberFilter(label="Filter pk", method="incident_filter")
```

Comment:
To avoid the weird NumberFilter/decimal issue with django-filter, I propose defining this custom class first:

```python
class IntegerFilter(filters.NumberFilter):
    field_class = forms.IntegerField
```

Then changing the above line to 
```suggestion
    filter_pk = filters.IntegerFilter(label="Filter pk", method="incident_filter")
```

That makes the tests pass too, easy peasy.